canonical/ubuntu.com
# Add pages to install and manage lxd

## Done

- added lxd/install and lxd/manage pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- check /lxd, /lxd/install and /lxd/manage

## Issue / Card

Fixes WD-4823
